templates/taxi/manufacturer_delete_confirm.html

```diff
@@ -0,0 +1,10 @@
+{% extends 'base.html' %}
+
+{% block content %}
+  <h1>Are you sure you want to delete "{{ object }}"</h1>

```

Comment:
Ensure that the `object` context variable is passed to the template from the view, as it is used here to display the name or identifier of the manufacturer being deleted.